py-scripts/real_application_tests/real_browser/lf_interop_real_browser_test.py

```diff
@@ -1509,32 +1985,32 @@ def main():
         parser.add_argument('--webgui_incremental', '--incremental_capacity', help="Specify the incremental values <1,2,3..>", dest='webgui_incremental', type=str)
         parser.add_argument('--incremental', help="to add incremental capacity to run the test", action='store_true')
         optional.add_argument('--no_laptops', help="run the test without laptop devices", action='store_false')
-        parser.add_argument('--postcleanup', help="Cleanup the cross connections after test is stopped", action='store_true')
-        parser.add_argument('--precleanup', help="Cleanup the cross connections before test is started", action='store_true')
+        parser.add_argument('--postcleanup', help="Cleanup the cross connections after test is stopped", action='store_true', default=True)
+        parser.add_argument('--precleanup', help="Cleanup the cross connections before test is started", action='store_true', default=True)
```

Comment:
Is the intention here to make the script always cleanup before and after the test? If so, perhaps we can change these to no cleanup and no post cleanup, similar to other scripts. That would make this script consistent with the others